clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/handlers/BasicGoogleDeployHandler.groovy

```diff
@@ -488,7 +492,7 @@ class BasicGoogleDeployHandler implements DeployHandler<BasicGoogleDeployDescrip
         .setTargetPools(targetPools)
         .setAutoHealingPolicies(autoHealingPolicy)
 
-    if (hasBackendServices && (description?.loadBalancingPolicy || description?.source?.serverGroupName))  {
+    if (!internalLoadBalancers && (description?.loadBalancingPolicy || description?.source?.serverGroupName))  {
```

Comment:
As it's written, lbs that isn't `internal` (= external http/tcp/ssl , and internal HTTP)  requires namedPorts handling.
Because `hasBackendServices` is false when internal HTTP loadbalancers exists, now it's not appropriate here. (`hasBackendServices` should be renamed to `hasGlobalBackendServices`)

This condition can be written as following way to reduce the change impact, but I think it's not clean.
`if ((hasBackendServices || internalHttpLoadBalancers) && (description?.loadBalancingPolicy || description?.source?.serverGroupName))`

Reply:
Thanks for the explanation! Isn't it possible that we have `internalLoadBalancers` _and_ `tcpLoadsBalancers` (i.e. two types of load balancers for this server group)? In which case we need to configure the named ports, but as it's written here we won't?

Reply:
Wow that's right.
- [ ] fix